components/pedigree/resources/src/main/resources/pedigree/controller.js

```diff
@@ -496,7 +496,7 @@ define([
                 editor.getUndoRedoManager().addState( event, undoEvent );
             }
 
-            editor.getNodeMenu().update();  // for example, user selected a wrong gender in the nodeMenu, which
+            ((node.getType() == 'PersonGroup') ? editor.getNodeGroupMenu() : editor.getNodeMenu()).update();  // for example, user selected a wrong gender in the nodeMenu, which
```

Comment:
we should improve it and make it more object-oriented - e.g. a `node` should know which "node menu" it uses (which it does, as it creates it, just does not have a way to report it as of right now) instead of using an explicit `if/else`, but that can be a `TODO`